gsad/src/gsad_gmp.c

```diff
@@ -2014,12 +1598,12 @@ get_many (gvm_connection_t *connection, const char *type,
           else
             filt_id = FILT_ID_USER_SETTING;
         }
-      else if ((strcmp (filter, "sort=nvt") == 0)
-               && ((strcmp (type, "note") == 0)
-                   || (strcmp (type, "override") == 0)))
+      else if (str_equal (filter, "sort=nvt")
+               && (str_equal (type, "note")
+                   || str_equal (type, "override") == 0))
```

Comment:
It looks like you accidentally left the == 0 in here.

Reply:
You are right. I'll fix it.